### ISSUE TITLE
A11y_code.visualstudio.com_VisualStudioCode_Download_Platform Specific_AI4D: Name property is missing for 'Visual studio code in action' list items.

### ISSUE THREAD

**Issue by bhagwatshelke1047** [microsoft/vscode-docs#3716]
[Check out Accessibility Insights!](url) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

GitHubTags:#A11y_Code.visualstudio.com_Benchmark;#A11ySev2;#A11yTCS;#A11yMAS;#Benchmark;#ChromiumEdge;#DesktopWeb;#MAS4.1.2;#Win10;#Visual Studio Code;#AI4D;#CAITestPass_May2020;FTP

**Environment Details:**
Application Name: code.visualstudio.com
#URL:[https://code.visualstudio.com](url)
Windows Version: Windows10
Browser version: Microsoft Edge 84.0.488.1 (Official build) dev (64-bit)

**Repro Steps:**
1. Launch the application Url :[https://code.visualstudio.com](url)
2. Navigate to "Download" button.
3. Hit on any of the platform(Windows/Linux/MacOS) links or any installer type(*64 and *32).
5. Start AI4D and check for 'Visual studio code list items'.

**Actual:**
Name property is missing for 'Visual studio code in action' list items.

**Expected:**
Name property should be there  for 'Visual studio code in action' list items.

Attachment  for Reference:
![image](https://user-images.githubusercontent.com/52443697/82981984-d57bd100-a00a-11ea-8a53-edc1380e9eff.png)

**Comment by fsteffi** [comment]
issue has been fixed in https://code.visualstudio.com/docs/?dv=linux64_deb
![image](https://user-images.githubusercontent.com/24580570/216556344-e68af0f5-aa7a-4490-a007-4504c0280e01.png)

**GithubTags:**#Closed;
